packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -232,6 +253,35 @@ export class PluginLedgerConnectorCorda
       endpoints.push(endpoint);
     }
 
+    {
+      const opts: IListCPIEndpointV1Options = {
```

Comment:
We will also need to enable/disable these endpoints based on the version specified by `CordaVersion` var type